frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -511,7 +511,7 @@ export default function SurveyEdit(): JSX.Element {
                             : []),
                         {
                             key: SurveyEditSection.DisplayConditions,
-                            header: 'Display conditions',
+                            header: <div data-attr="survey-display-conditions">Display conditions</div>,
```

Comment:
Not sure if there's a more preferable option for targeting this - the div wrap has no visual effect. Could go with a more brittle cypress `contains('Display conditions')` instead, or add formal support for `id`'s or `data-attr`'s in `LemonCollapse`

Reply:
Yeah, this is fine, if a bit ugly – even thought `header` can be a ReactNode, we basically just pass it strings every time.  Would be cool if `LemonCollapse` support `data-attrs` instead; how hard would that be?

Reply:
Yup, not too bad to do that, sounds good to me

Reply:
done!